go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/launchdarkly/go-configtypes
 go 1.13
 
 require (
+	github.com/alecthomas/units v0.0.0-20231202071711-9a357b53e9c9 // indirect
```

Comment:
Wonder why this is labeled as `indirect` even though it is a direct dependency 🤔

Reply:
Another go mod tidy seemed to fix that.